### PR TITLE
Initial contributing guides

### LINKED ISSUE #4: Create a contributing guideline document
Some bits to consider are:

1. How best to share common guidelines across the multiple toolkits.
2. How best to split out the toolkit specific parts (would this be best in multiple documents?)
3. As per this [comment](https://github.com/MicrosoftDocs/CommunityToolkit/pull/2#issuecomment-1024344199) for the .NET MAUI Community Toolkit
    > it should always be .NET MAUI. Never just MAUI. Then, on first use on every page it should always be .NET Multi-platform App UI (.NET MAUI).
4. Other points? please feel free to comment

### PR DESCRIPTION
**Pull request by bijington**
Adds an initial contributing guideline document for the .NET MAUI Community Toolkit and a top level one to link between the specific guides.

Fixes #4

### CODE REVIEW

**Comment by bijington** [comment]
A bulk review request to see what everyone thought about having the top level guide linking down to specific ones.

@Michael-hawker do you want me to add anything in to cover the Windows and .NET toolkits for now?

@davidbritch what would be the best way of storing the specific guidelines files without having them published? Should I just go with the `.` prefix on the filename or is there a better option here?

**Comment by davidbritch** [comment]
@bijington Personally I'd put any contrib docs outside of the docs folder. So maybe create a root folder to store them?

**Comment by bijington** [comment]
> @bijington Personally I'd put any contrib docs outside of the docs folder. So maybe create a root folder to store them?

**Comment by bijington** [comment]
Great idea thanks!

**Comment by PRMerger12** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Repository root file |  This pull request contains a file submitted to the root of the repository.

**Comment by PRMerger5** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Repository root file |  This pull request contains a file submitted to the root of the repository.

**Comment by PRMerger20** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Repository root file |  This pull request contains a file submitted to the root of the repository.

**Review by jfversluis — APPROVED** [review]
Seems to make sense to me, does @davidbritch have any professional comments to add here?

File: contributing/contributing-maui.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Contribute to the .NET MAUI Community Toolkit documentation
+
+Thank you for your interest in contributing to the .NET MAUI Community Toolkit documentation.
+
+Documentation is **required** when completed a proposal as part of our [proposal workflow](https://github.com/CommunityToolkit/Maui/projects).
```
Comment (davidbritch) [inline contributing/contributing-maui.md:5]: "when completing a"...

or

"when you've completed a"....

File: contributing/contributing-maui.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Documentation is **required** when completed a proposal as part of our [proposal workflow](https://github.com/CommunityToolkit/Maui/projects). 
+
+## Starting your changes
+
+You will need to fork this repository. We strongly recommend that you create a branch from your `main` branch so that it will reduce any complications when submitting your Pull Request.
+
+## Recommended tooling
+
+We recommend that you use Visual Studio Code for the editing of documentation files along with the [Docs Authoring Pack](https://marketplace.visualstudio.com/items?itemName=docsmsft.docs-authoring-pack) extension. This helps with validating code example formatting and also spell checking among other rules, for more information check out the documentation at: https://docs.microsoft.com/contribute/how-to-write-docs-auth-pack
```
Comment (davidbritch) [inline contributing/contributing-maui.md:13]: Make the final link a markdown link?

Comment (davidbritch) [inline contributing/contributing-maui.md:13]: Make the final link a markdown link?

**Comment by bijington** [comment]
Thanks @davidbritch

**Comment by PRMerger5** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Repository root file |  This pull request contains a file submitted to the root of the repository.

**Review by brminnick — APPROVED** [review]

**Comment by PRMerger5** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Repository root file |  This pull request contains a file submitted to the root of the repository.

**Pull request merged**